pkg/controller/clusterconnection/clusterconnection_controller.go

```diff
@@ -191,6 +187,7 @@ func (r *ReconcileConnection) Reconcile(ctx context.Context, request reconcile.R
 	if err != nil {
 		return result, err
 	}
+	isCalicoEnterprise := variant == operatorv1.TigeraSecureEnterprise
```

Comment:
This is really nitty, but I think we should just use `variant == operatorv1.TigeraSecureEnterprise` throughout instead of aliasing it as a new variable. This is consistent with how we do this same check in other similar controllers.